include/hermes/VM/HandleRootOwner.h

```diff
@@ -98,7 +98,7 @@ class HandleRootOwner {
   /// An efficient way to pass empty to a function accepting Handle.
   static Handle<HermesValue> getEmptyValue();
 
-  /// An efficient way to pass bools to a function accepting Handle.
```

Comment:
This doesn't seem like a typo

Reply:
Please compare with other comments in the same file. They all use a singular form. I would suggest keeping the change for consistency.

Reply:
The others are talking about specific values like 0 or undefined, whereas this is talking about a type. If this were specifically "true" or "false", I think singular would be fine, but given that we are talking about a type, I think it should be "pass bools" or "pass a bool".

Reply:
It makes sense. It was reverted. Thank you!